app/services/licences/fetch-licence-version-purpose-condition.service.js

```diff
@@ -10,45 +10,74 @@ const LicenceVersionPurposeCondtionModel = require('../../models/licence-version
 /**
  * Fetch the matching licence version purpose conditions and return data needed for the view licence page
  *
- * @param {string} id The UUID for the licence_version_purpose_id to fetch
+ * @param {object} ids The object contains the UUIDs for the licence_version_purpose_id and the purpose_id
  *
  * @returns {Promise<Object>} the data needed to populate the view licence page and some elements of the summary tab
  */
-async function go (id) {
-  if (!id) {
+async function go (ids) {
+  if (!ids) {
     return {
-      abstractionConditions: []
+      numberOfAbstractionConditions: 0,
+      uniqueAbstractionConditions: []
     }
   }
-  const LicenceVersionPurposeCondition = await _fetchLicenceVersionPurposeCondition(id)
-  const data = await _data(LicenceVersionPurposeCondition)
+
+  const licenceVersionIds = []
+
+  ids.forEach((item) => { licenceVersionIds.push(item.licenceVersionPurposeId) })
+
+  const LicenceVersionPurposeCondition = await _fetchLicenceVersionPurposeCondition(licenceVersionIds)
+  const data = await _data(LicenceVersionPurposeCondition, ids)
 
   return data
 }
 
-async function _data (licenceVersionPurposeConditions) {
+async function _data (licenceVersionPurposeConditions, licenceVersionPuroseAndPurposeUseIds) {
   const abstractionConditions = []
 
   if (!licenceVersionPurposeConditions) {
-    return abstractionConditions
+    return {
+      numberOfAbstractionConditions: 0,
+      uniqueAbstractionConditions: []
+    }
   }
 
-  licenceVersionPurposeConditions.forEach((purposeConditions) => {
-    purposeConditions.licenceVersionPurposeConditionTypes.forEach((licenceVersionPurposeConditionType) => {
-      abstractionConditions.push(licenceVersionPurposeConditionType.displayTitle)
+  licenceVersionPurposeConditions.forEach((purposeCondition) => {
+    // Get the purposeUseId from the LicenceVersionPurpose table using the licenceVersionPurposeId of the current purposeCondition
+    const { purposeId: purposeUseId } = licenceVersionPuroseAndPurposeUseIds.find((item) =>
+      item.licenceVersionPurposeId === purposeCondition.licenceVersionPurposeId)
```

Comment:
We have a team convention that states we always use block body over concise for [arrow functions](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#arrow-functions) even if it is a one-liner.
```suggestion
    const { purposeId: purposeUseId } = licenceVersionPurposeAndPurposeUseIds.find((item) => {
      return item.licenceVersionPurposeId === purposeCondition.licenceVersionPurposeId
    })
```